src/frontend/utils/theme/theme.ts

```diff
@@ -82,13 +84,21 @@ const colorsMain = {
   white: '#ffffff',
 };
 
-const chatFonts = {
+interface Font {
+  color?: ColorType;
+  fontFamily?: string;
+  fontSize?: TextProps['size'];
+  fontWeight?: TextProps['weight'];
+  letterSpacing?: string;
+  lineHeight?: string;
+}
+
+const chatFonts: { [key: string]: Font } = {
```

Comment:
```suggestion
const chatFonts: { 
  primary: Font,
  secondary: Font,
  tertiary: Font
} = {
```

It's more redundant but also more accurate because I can write in my code `chatFonts.foo.color` and it's valid

Reply:
That is an interesting thing to notice. I totally agree with you, this is more convenient when typescrit doesn't type non-defined objects.
done in commit [0ad8a4](https://github.com/openfun/marsha/pull/1247/commits/0ad8a4df10ed0085eaa0c64b90e320ffd8bf139e)